### ISSUE TITLE
Unable to Alter Column when Altering from/to Encrypted Column

### ISSUE THREAD

**Issue by saklanipankaj** [aprismatic/prismadb#7]
```
CREATE TABLE t3
(
    a INT,
    b INT8
);
ALTER TABLE t3 ALTER COLUMN
a INT ENCRYPTED FOR(MULTIPLICATION, ADDITION, SEARCH) NULL;
```

> ERROR:  Altering of encryption types using AlterTableQuery has been deprecated. Use EncryptColumnCommand or DecryptColumnCommand instead.

Altering Table Column from/to Encrypted throws an internal code error.

**Comment by cheziyi** [comment]
As mentioned in https://github.com/PrismaDB/PrismaDB/wiki/ALTER-TABLE, for altering of encryption, you will need to use `PRISMADB ENCRYPT` or `PRISMADB DECRYPT` commands instead for security reasons. That error message meant exactly this. We will update the naming of the queries in the error message to be less confusing to the user.

**Comment by cheziyi** [comment]
Fixed in MySQL 0.3.3.75, SQL Server 0.3.3.102, PostgreSQL 0.3.3.37
